packages/gatsby-telemetry/src/telemetry.js

```diff
@@ -71,9 +85,21 @@ module.exports = class AnalyticsTracker {
     }
 
     const decoration = this.metadataCache[type]
-    delete this.metadataCache[type]
     const eventType = `${baseEventType}_${type}`
-    this.buildAndStoreEvent(eventType, Object.assign(tags, decoration))
+
+    if (debounce.debounce) {
```

Comment:
Can't this just be `debounce` as opposed to `debounce.debounce`?

Reply:
Updated, renamed the params object to opts for future expansibility.